diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/
```

Comment:
Almost, the full path would be 
```sh
_/Users/grace/wdi/try_
```
It you look at the question, it has you create wdi, and then we cd into it. Then create try directory and cd into that. Making the path above.